### ISSUE TITLE
Parse logs on Varnish!

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/data-api#460]

**Comment by BobTorgerson** [comment]
Agreed! We had looked into a combination of Loki, Prometheus, and Grafana, but the results was overly complex for the results that it provides for us. I think we can park this as mostly unnecessary for now, but a simple parsing solution looking for URLs and error codes that stem from those are really all we need.
